modules/oc-mirror-command-reference.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: REFERENCE
+[id="oc-mirror-command-reference_{context}"]
+= Command reference for oc-mirror
+
+The following tables describe the `oc mirror` subcommands and flags:
+
+.oc mirror subcommands
+[cols="1,2",options="header"]
+|===
+|Subcommand
+|Description
+
+|`completion`
+|Generate the autocompletion script for the specified shell.
+
+|`describe`
+|Output the contents of an image set.
+
+|`help`
+|Show help about any subcommand.
+
+|`list`
+|List available platform and Operator content and their version.
+
+|`version`
+|Output the oc-mirror version.
+
+|===
+
+.oc mirror flags
+[cols="1,2",options="header"]
+|===
+|Flag
+|Description
+
+|`-c`, `--config` `<string>`
+|Specify the path to an image set configuration file.
+
+|`--continue-on-error`
+|If any non image-pull related error occurs, continue and attempt to mirror as much as possible.
+
+|`--dest-skip-tls`
+|Disable TLS validation for the target registry.
+
+|`--dest-use-http`
+|Use plain HTTP for the target registry.
+
+|`--dry-run`
+|Print actions without mirroring images. This flag is experimental and only works when mirroring to disk.
+
+|`--from <string>`
+|Specify the path to an image set archive that was generated by an execution of oc-mirror to load into a target registry.
+
+|`-h`, `--help`
+|Show the help.
+
+|`--log-level <string>`
+|Specify the log level, such as `debug`, `info`, `warn`, or `error`. The default value is `info`.
+
+|`--manifests-only`
+|Generate manifests for `ImageContentSourcePolicy` objects to configure a cluster to use the mirror registry, but do not actually mirror any images.
```

Comment:
`--manifests-only`  only works for an archive with --from, better to highlight it .

Reply:
I will create a separate PR to update this for all versions (since this PR is only for 4.10)

Reply:
Created #60888 for this followup item